site/sass/components/_toc.scss

```diff
@@ -3,22 +3,25 @@
 // Breakpoints are tied to tech-detail layout, so before reusing it some adjustments may be necessary.
 
 .toc {
-  @include expandingNav($bkpt-tech-details--3-cols, 'toc', 1rem, 1rem);
+  $shadow-color: rgba(map-get($google-colors, 'grey-900'), 0.1);
```

Comment:
Instead of constructing this with an alpha and a dark grey, can we use one of our light greys (200 or below) instead?

Reply:
Done.